src/Features/Core/Portable/AddParameter/AddParameterService.cs

```diff
@@ -101,7 +101,7 @@ public bool HasCascadingDeclarations(IMethodSymbol method)
                 var generator = editor.Generator;
                 foreach (var methodDeclaration in documentLookup)
                 {
-                    var methodNode = syntaxRoot.FindNode(methodDeclaration.Locations[0].SourceSpan);
+                    var methodNode = syntaxRoot.FindNode(methodDeclaration.Locations[0].SourceSpan, getInnermostNodeForTie: true);
```

Comment:
we might want to audit most of these.  we always knew and used getInnermostNodeForTie for expressions (because of teh argument/expression duality), but now we'll have this a lot for statements, so likely most calls are suspect :)